subprojects/docs/src/docs/release/notes.md

```diff
@@ -120,6 +120,14 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
+<a name="configuration-cache"></a>
+### Configuration Cache improvements
+The configuration cache is a feature that significantly improves build performance by caching the result of the configuration phase and reusing this for subsequent builds.
+
+#### New API in `FileSystemOperations` to create `CopySpec` instances when no `Project` is available
```

Comment:
❌   I would not add this under `Configuration cache improvements` but under a `Plugin authoring improvements`. It happens it helps with the configuration cache but it is helpful in general for plugins. For example it can now be used from an isolated worker action or an artifact transform because the `FileSystemOperations` service is injectable there!